### PR TITLE
Block details

### PR DESCRIPTION
**Pull request by PabloCastellano**
Fixes https://github.com/Blobscan/blobscan/issues/133

Changes:

* Do not format **Block and Slot number** with commas, as in other block explorers. It also makes it easier to copy-paste.
* **Blob Gas Price** now is shown in ether and gwei for easier conversions
* **Removed 'Total' from metric names**
* **Blob Gas Used** now shows the percentage of blob space used and target blob gas (reminder: maximum blobs per block is 6, while target is 3, similar to how the gas fee work)
* Show number of blobs next to **Blob Size**
* Show **Blob Gas Limit**

**BEFORE THIS PR:**

image

**AFTER THIS PR:**

image

### CODE REVIEW

File: apps/web/src/pages/block/[id].tsx [hunk 1]
```diff
@@ -96,24 +99,65 @@ const Block: NextPage = function () {
...
                 {
-                  name: "Total Blob Size",
-                  value: formatBytes(totalBlobSize),
+                  name: "Blob Size",
+                  value: (
+                    <div>
+                      {formatBytes(totalBlobSize)}
+                      <span className="ml-1 text-gray-500">
+                        ({formatNumber(BigInt(totalBlobSize) / GAS_PER_BLOB)}{" "}
+                        blobs)
```
[the remark this answers is not in the document]
Comment (PabloCastellano) [reply apps/web/src/pages/block/[id].tsx:113]: Do you mean that "blobs" is always plural here?

File: apps/web/src/pages/block/[id].tsx [hunk 2]
```diff
@@ -96,24 +99,65 @@ const Block: NextPage = function () {
...
+                  ),
                 },
                 {
                   name: "Blob Gas Price",
-                  value: formatWei(blockData.blobGasPrice),
+                  value: (
+                    <div>
+                      {formatWei(blockData.blobGasPrice, { toUnit: "ether" })}
+                      <span className="ml-1 text-gray-500">
+                        ({formatWei(blockData.blobGasPrice)})
```
[the remark this answers is not in the document]
Comment (PabloCastellano) [reply apps/web/src/pages/block/[id].tsx:124]: Agree, changed

File: apps/web/src/utils/ethereum.ts [hunk 3]
```diff
@@ -57,6 +60,20 @@ export function convertWei(
   }
 }
 
+export function gasTarget(blobGasUsed: bigint): string {
```
[the remark this answers is not in the document]
Comment (PabloCastellano) [reply apps/web/src/utils/ethereum.ts:63]: I don't have a strong opinion here

File: apps/web/src/pages/block/[id].tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (0xGabi) [reply apps/web/src/pages/block/[id].tsx:114]: Yes exactly.

File: apps/web/src/utils/ethereum.ts (hunk 3 above)
[the remark this answers is not in the document]
Comment (PJColombo) [reply apps/web/src/utils/ethereum.ts]: I agree with Gabi on this one.

Let's follow the Single Responsibility Principle and separate both computing and formatting logic

File: apps/web/src/pages/block/[id].tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (PJColombo) [reply apps/web/src/pages/block/[id].tsx]: Resolved!

**Review by PJColombo — APPROVED** [review]
Great job 👍!

Added some extra refactorizactions and resolved the remaining review conversations

**Pull request merged**